packages/wrangler/src/pages/index.tsx

```diff
@@ -83,8 +83,9 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
 							Publish.Handler
 						)
 						.command(
-							"tail [deployment-id]",
-							"Tail requests and logs on a deployment",
+							"tail [deployment]",
+							"Start a tailing session for a project's deployment and " +
+								" livestream logs from your Functions",
```

Comment:
remove white space. There's already a trailing one on the line above 👇
<img width="741" alt="Screenshot 2022-11-11 at 11 57 08" src="https://user-images.githubusercontent.com/4638332/201326638-fa6bbf72-8bc6-46e2-92ae-45d79eca71ef.png">

Reply:
Done